src/install/kafka/whatsNext.mdx

```diff
@@ -0,0 +1,4224 @@
+      ' '
+    }
+
+    <tr>
+      <td>
+        **SASL_PASSWORD**
+      </td>
+
+      <td>
+        SASL password required with the PLAIN and SCRAM mechanisms.
+      </td>
+
+      <td>
+        N/A
+      </td>
+
+      <td style={{ 'text-align': 'center' }}>
+        M/I
+      </td>
+    </tr>
+
+    {
+      ' '
+    }
+
+    <tr>
+      <td>
+        **SASL_GSSAPI_REALM**
+      </td>
+
+      <td>
+        Kerberos realm required with the GSSAPI mechanism.
+      </td>
+
+      <td>
+        N/A
+      </td>
+
+      <td style={{ 'text-align': 'center' }}>
+        M/I
+      </td>
+    </tr>
+
+    {
+      ' '
+    }
+
+    <tr>
+      <td>
+        **SASL_GSSAPI_SERVICE_NAME**
+      </td>
+
+      <td>
+        Kerberos service name required with the GSSAPI mechanism.
+      </td>
+
+      <td>
+        N/A
+      </td>
+
+      <td style={{ 'text-align': 'center' }}>
+        M/I
+      </td>
+    </tr>
+
+    {
+      ' '
+    }
+
+    <tr>
+      <td>
+        **SASL_GSSAPI_USERNAME**
+      </td>
+
+      <td>
+        Kerberos username required with the GSSAPI mechanism.
+      </td>
+
+      <td>
+        N/A
+      </td>
+
+      <td style={{ 'text-align': 'center' }}>
+        M/I
+      </td>
+    </tr>
+
+    {
+      ' '
+    }
+
+    <tr>
+      <td>
+        **SASL_GSSAPI_KEY_TAB_PATH**
+      </td>
+
+      <td>
+        Kerberos key tab path required with the GSSAPI mechanism.
+      </td>
+
+      <td>
+        N/A
+      </td>
+
+      <td style={{ 'text-align': 'center' }}>
+        M/I
+      </td>
+    </tr>
+
+    {
+      ' '
+    }
+
+    <tr>
+      <td>
+        **SASL_GSSAPI_KERBEROS_CONFIG_PATH**
+      </td>
+
+      <td>
+        Kerberos config path required with the GSSAPI mechanism.
+      </td>
+
+      <td>
+        /etc/krb5.conf
+      </td>
+
+      <td style={{ 'text-align': 'center' }}>
+        M/I
+      </td>
+    </tr>
+
+    {
+      ' '
+    }
+
+    <tr>
+      <td>
+        **SASL_GSSAPI_DISABLE_FAST_NEGOTIATION**
+      </td>
+
+      <td>
+        Disable FAST negotiation.
+      </td>
+
+      <td>
+        false
+      </td>
+
+      <td style={{ 'text-align': 'center' }}>
+        M/I
+      </td>
+    </tr>
+  </tbody>
+</table>
+
+## Broker Collection filtering [#broker-filteri]
+
+<table>
+  <thead>
+    <tr>
+      <th style={{ width: "150px" }}>
+        Setting
+      </th>
+
+      <th>
+        Description
+      </th>
+
+      <th>
+        Default
+      </th>
+
+      <th>
+        Applies To
+      </th>
+    </tr>
+  </thead>
+
+  <tbody>
+    <tr>
+      <td>
+        **LOCAL_ONLY_COLLECTION**
+      </td>
+
+      <td>
+        Collect only the metrics related to the configured bootstrap broker. Only used if `autodiscover_strategy` is `bootstrap`.
+
+        **Environments that use discovery (such as Kubernetes) must be set to true because othwerwise brokers will be discovered twice: By the integration, and by the discovery mechanism, leading to duplicate data.**
```

Comment:
You must set environments that use discovery (such as Kubernetes) to true, otherwise it will discover brokers twice:

I'm not sure what the 'it' should be above, but you can fix the passive voice this way.